a11y-speak.js

```diff
@@ -76,6 +76,8 @@ var A11ySpeak = function( message, ariaLive ) {
 	// Clear previous messages to allow repeated strings being read out.
 	clear();
 
+	message = message.replace( /<[^<>]+>/g, "" );
```

Comment:
Ideally the A11ySpeak messages should be carefully crafted to be simple and specific to be used for A11ySpeak. I mean, generally speaking, re-using existing messages which contain HTML is not a good idea in the first place.
But, reusing strings can be handy in some circumstances so I'd consider this something to keep simple just in case there's some HTML. We could replace with a space instead of an empty string but maybe that would be just an assumption. It would be OK for cases like this:
`<span>first</span><span>second</span>`

but not OK for cases like this:
`<span>just part of a word is high<b>light</b>ed</span>`

I'd lean towards keeping it as is. Maybe add a comment :)